inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/elasticsearch/EsIdConfig.java

```diff
@@ -86,6 +88,7 @@ public static EsIdConfig create(DataFlowConfig dataFlowConfig) {
                 .separator(sinkConfig.getSeparator())
                 .indexNamePattern(sinkConfig.getIndexNamePattern())
                 .fieldList(fields)
+                .charset(Charset.forName(sinkConfig.getEncodingType()))
```

Comment:
It maybe throw Exception for Charset.forName.
It is better to use default charset.

Reply:
fixed, thx